App/Sagas/TextileSagas.js

```diff
@@ -102,6 +103,9 @@ export function * handleNewAppState ({previousState, newState}) {
     } else if (previousState.match(/inactive|active/) && newState === 'background') {
       console.tron.logImportant('app transitioned to background')
       yield * triggerStopNode()
+    } else if (previousState.match(/active/) && newState === 'active') {
```

Comment:
so basically, we can call triggerCreateNode() almost be default any time we have a stateTransition, and just catch the one case where we are backgrounding right?

Reply:
this one might not even be needed